manager/integration/tests/common.py

```diff
@@ -1677,9 +1677,10 @@ def wait_for_volume_faulted(client, name):
                                   VOLUME_ROBUSTNESS_FAULTED)
 
 
-def wait_for_volume_status(client, name, key, value):
+def wait_for_volume_status(client, name, key, value,
+                           timeout_in_sec=RETRY_COUNTS//2):
```

Comment:
Why not just 

`def wait_for_volume_status(client, name, key, value, timeout_in_sec=RETRY_COUNTS):`

Reply:
Since the wait interval is 0.5s rather than 1s.
I don't want to affect other test cases or slow down the status check/wait since I choose not to modify the interval.